autosklearn/evaluation/__init__.py

```diff
@@ -1,5 +1,18 @@
 # -*- encoding: utf-8 -*-
-from typing import Any, Callable, Dict, List, Optional, Tuple, Type, Union, cast
+from __future__ import annotations
+
+from typing import (
```

Comment:
why this?

Reply:
Because I added one or two more imports, black decided that the line is too long.